boards/arm/frdm_k64f/dts_fixup.h

```diff
@@ -4,12 +4,12 @@
 #define DT_FXOS8700_GPIO_NAME		DT_NXP_KINETIS_I2C_40066000_NXP_FXOS8700_1D_INT2_GPIOS_CONTROLLER
 #define DT_FXOS8700_GPIO_PIN		DT_NXP_KINETIS_I2C_40066000_NXP_FXOS8700_1D_INT2_GPIOS_PIN
 
-#define CONFIG_IEEE802154_MCR20A_SPI_DRV_NAME		DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_BUS_NAME
-#define CONFIG_IEEE802154_MCR20A_SPI_SLAVE		DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_BASE_ADDRESS
-#define CONFIG_IEEE802154_MCR20A_SPI_FREQ		DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_SPI_MAX_FREQUENCY
-#define CONFIG_IEEE802154_MCR20A_GPIO_SPI_CS_DRV_NAME	DT_NXP_KINETIS_DSPI_4002C000_CS_GPIOS_CONTROLLER
-#define CONFIG_IEEE802154_MCR20A_GPIO_SPI_CS_PIN	DT_NXP_KINETIS_DSPI_4002C000_CS_GPIOS_PIN
-#define CONFIG_MCR20A_GPIO_IRQ_B_NAME			DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_IRQB_GPIOS_CONTROLLER
-#define CONFIG_MCR20A_GPIO_IRQ_B_PIN			DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_IRQB_GPIOS_PIN
-#define CONFIG_MCR20A_GPIO_RESET_NAME			DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_RESET_GPIOS_CONTROLLER
-#define CONFIG_MCR20A_GPIO_RESET_PIN			DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_RESET_GPIOS_PIN
+#define DT_IEEE802154_MCR20A_SPI_DRV_NAME		DT_NXP_KINETIS_DSPI_4002C000_NXP_MCR20A_0_BUS_NAME
```

Comment:
should we move those out of the board? frdm_k64f does not have an MCR20A AFAIK.

Reply:
Yes, can we do it as a follow up PR.  Because soc/arm/nxp_kinetis/kwx/dts_fixup.h also needs addressing.

Reply:
> soc/arm/nxp_kinetis/kwx/dts_fixup.h

afaik this guy actually has a radio

Reply:
@MaureenHelm is that correct?

Reply:
Now that you mention it, that sounds familiar.

Reply:
Either way, the pull out of MCR20A from the frdm_k64f.dts I'll handle in a followup PR.

Reply:
We have issue #7462 for it.

Reply:
ok